plugins/mygene/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@jbrowse/plugin-mygene",
+  "version": "1.0.2",
+  "description": "JBrowse 2 mygene.",
+  "keywords": [
+    "jbrowse",
+    "jbrowse2"
+  ],
+  "license": "Apache-2.0",
+  "homepage": "https://jbrowse.org",
+  "bugs": "https://github.com/GMOD/jbrowse-components/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GMOD/jbrowse-components.git",
+    "directory": "plugins/mygene"
+  },
+  "author": "JBrowse Team",
+  "distMain": "dist/index.js",
+  "srcMain": "src/index.ts",
+  "main": "src/index.ts",
+  "distModule": "dist/plugin-mygene.esm.js",
+  "module": "",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "scripts": {
+    "start": "tsdx watch --verbose --noClean",
+    "build": "tsdx build",
+    "test": "cd ../..; jest plugins/mygene --passWithNoTests",
+    "prepublishOnly": "yarn test",
+    "prepack": "yarn build; yarn useDist",
+    "postpack": "yarn useSrc",
+    "useDist": "node ../../scripts/useDist.js",
+    "useSrc": "node ../../scripts/useSrc.js"
+  },
+  "dependencies": {
+    "@gmod/gff": "^1.1.2",
+    "@gmod/tabix": "^1.4.6",
+    "abortable-promise-cache": "^1.4.0",
+    "generic-filehandle": "^2.0.0"
+  },
+  "peerDependencies": {
+    "@jbrowse/core": "^1.0.0",
+    "@jbrowse/plugin-linear-genome-view": "^1.0.0",
+    "@material-ui/core": "^4.9.13",
+    "mobx-react": "^6.0.0",
+    "mobx-state-tree": "3.14.1",
+    "prop-types": "^15.0.0",
+    "react": "^16.8.0",
+    "rxjs": "^6.0.0"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "gitHead": "32e27bccb4eb35ee2b3ba3ab004baf68dfcbf38d"
```

Comment:
I think gitHead is added transiently when publishing and then removed. Looks like this may have been copied over from `plugins/gff3/package.json`? It should probably be removed in both places.

Reply:
removed this from both gff3 and here